src/components/Home.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Home = () => (
+  <section>
+    <div className="container">
+      <h2>Welcome to the math world!</h2>
+      <p>Lets play around numbers.</p>
+    </div>
+  </section>
+
+);
+
+export default Home;
```

Comment:
- [ ] Please make sure that the general layout of your `Quotes` page matches the given wireframe in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  As you can see in the wireframe the home page is left aligned at begin from the start of the page.

|Your design |wireframe design|
|------|------|
|<img width="1154" alt="Screenshot 2023-06-14 at 9 42 15 AM" src="https://github.com/rabayarabu/calculator/assets/44798044/329b592e-dbb2-4588-8de8-657822e9574f">|<img width="1054" alt="Screenshot 2023-06-14 at 9 30 55 AM" src="https://github.com/rabayarabu/calculator/assets/44798044/dd7e386c-db25-49e2-84f7-8c4fa11444b4">|

**NOTE:** Students are encouraged to add their styles to improve the look and feel.